src/components/molecules/PieChartView.tsx

```diff
@@ -1,25 +1,26 @@
 import React, { FunctionComponent } from 'react'
 import { VictoryPie, VictoryContainer } from 'victory'
+import { IWidgetCountBySeverity } from '../../models/WidgetData'
 
 interface IProps {
-	data: pieChartDataType[]
-}
-export type pieChartDataType = {
-	accident_type: string
-	count: number
+	data: IWidgetCountBySeverity[] | any
+	x: string
+	y: string | number
 }
+
 export const PieChartView: FunctionComponent<IProps> = props => {
 	const pieChartViewStyle = {
 		labels: {
 			fontSize: 8,
 			fill: 'black'
 		}
 	}
+
 	return (
 		<VictoryPie
 			data={props.data}
-			x='accident_type'
-			y='count'
+			x={props.x}
```

Comment:
Are `x` and `y` the names you receive from the API?
I think it should use a more informative naming, such as `xLabel`.